glean-core/src/metrics/counter.rs

```diff
@@ -25,10 +28,14 @@ impl MetricType for CounterMetric {
 }
 
 impl CounterMetric {
+    /// Create a new metric
     pub fn new(meta: CommonMetricData) -> Self {
         Self { meta }
     }
 
+    /// Increase the counter by `amount`
+    ///
```

Comment:
Can we document arguments here, using the format we're using in other parts of the code?